src/project/index.js

```diff
@@ -0,0 +1,92 @@
+window.c.project.Index = (function(m, c){
```

Comment:
I think we should move this to the Pages namespace. Then we could abstract all the data logic we have on the controller to a view-model on a separated file.